apps/api/src/app/topics/use-cases/create-topic.use-case.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
I think that in other usecases we don't add a '-' in the usecase name. Wdyt?

Reply:
Ah, it is because I have seen a mix of different writings of use case with or without space, so I went for the technical term that is with space. Should I change to `usecase` then?